Cargo.toml

```diff
@@ -17,8 +17,8 @@ failure = "0.1"
 common_failures = "0.1"
 structopt = "0.3"
 env_logger = { version = "0.6.2", default-features = false, features = []}
-log = "*"
-crev-recursive-digest = { version = "0.2.1" }
+log = "0.4"
+crev-recursive-digest = { version = "0.4.0" }
```

Comment:
I've had to add permissions handling to this library. See https://crates.io/crates/crev-recursive-digest